packages/ngworker/lumberjack/console-driver/src/lib/configuration/lumberjack-console-driver.module.ts

```diff
@@ -18,13 +20,20 @@ export class LumberjackConsoleDriverModule {
    *
    * @param config Settings used by the console driver.
    */
-  static forRoot(config?: LumberjackConsoleDriverConfig): ModuleWithProviders<LumberjackConsoleDriverRootModule> {
+  static forRoot(
+    config: Partial<LumberjackConsoleDriverConfig> = {}
+  ): ModuleWithProviders<LumberjackConsoleDriverRootModule> {
     return {
       ngModule: LumberjackConsoleDriverRootModule,
       providers: [
         {
           provide: lumberjackConsoleDriverConfigToken,
-          useValue: config || {},
+          deps: [lumberjackLogDriverConfigToken],
+          useFactory: (logDriverConfig: LumberjackLogDriverConfig) => ({
```

Comment:
suggestion: add explicit return type to avoid type errors.
```suggestion
          useFactory: (logDriverConfig: LumberjackLogDriverConfig): LumberjackLogDriverConfig => ({
```